lib/version_utils.pm

```diff
@@ -973,3 +974,12 @@ sub is_community_jeos {
     return (get_var('FLAVOR', '') =~ /JeOS-for-(AArch64|RISCV|RPi)/);
 }
 
+=head2 has_selinux_by_default
+
+Returns true if the distro has SELinux as default MAC
+=cut
+
+sub has_selinux_by_default {
+    return is_tumbleweed || is_micro;
```

Comment:
This needs to be conditional for the TW staging with the skelcd change for now

Reply:
doesn't that make it harder to run iso tests on it or am i mistaken? 
also iirc the skelcd change was unstaged by ana

Reply:
nevermind, i added `is_staging`

Reply:
i will ask ana if she can stage it again

Reply:
okay she did in Staging:D, changed it